src/test/cypress/integration/Authentication.spec.ts

```diff
@@ -1,17 +1,14 @@
-/// <reference types="cypress" />
-
 import { authTokenKey } from '../support/constants';
+import { CypressUserManagement } from '../support/users';
+
+// The user management object
+const users = new CypressUserManagement();
+const user = users.getStudentOne();
 
-let username = Cypress.env('username');
-let password = Cypress.env('password');
-if (Cypress.env('isCi')) {
-    username = username.replace('USERID', '1');
-    password = password.replace('USERID', '1');
-}
 describe('Authentication tests', () => {
     beforeEach(() => {
-        expect(username, 'username was set').to.be.a('string').and.not.be.empty;
-        expect(password, 'password was set').to.be.a('string').and.not.be.empty;
+        expect(user.username, 'username was set').to.be.a('string').and.not.be.empty;
+        expect(user.password, 'password was set').to.be.a('string').and.not.be.empty;
```

Comment:
"<insert username here" is a string therefore this check does nothing. I would just remove it.

Reply:
You are right. With the user management changes this became obsolete, so I removed it